modules/collect/src/test/java/com/opengamma/strata/collect/result/PartialResultTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.collect.result;
+
+import static com.opengamma.strata.collect.TestHelper.assertSerialization;
+import static com.opengamma.strata.collect.TestHelper.coverBeanEquals;
+import static com.opengamma.strata.collect.TestHelper.coverImmutableBean;
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+
+import com.google.common.collect.ImmutableList;
+
+/**
+ * Test {@link PartialResult}.
+ */
+@Test
+public class PartialResultTest {
```

Comment:
The expectation is that the success value is a list, set or map capable of holding zero entries. Doing otherwise seems to complicate the state further.